portal/celery_worker.py

```diff
@@ -49,3 +49,11 @@ def setup_periodic_tasks(sender, **kwargs):
                                          task.s(*args_in, **kwargs_in))
             except Exception as exc:
                 logger.error(exc)
+
+    # add the healthcheck task
+    logger.info("Scheduling celery beat health check")
+    sender.add_periodic_task(
```

Comment:
@pbugni Should this be persisted in ScheduledJobs.json like the others?

Reply:
Yes, for consistency, I think it should.
@drryanjames ask if that doesn't make sense - you'll note there's a `ScheduledJobs.json` file for each application branch.  Bring into the system via `flask sync`.

Reply:
@pbugni @ivan-c will do. During my first pass I actually used ScheduledJobs.json but thought cron tabs were only executed once a day. I should have read the docs more carefully. Will update.